bin/wgwrangler.pl

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env perl
 
 use lib qw(); # PERL5LIB
-use FindBin;use lib "$FindBin::RealBin/../lib";use lib "$FindBin::RealBin/../thirdparty/lib/perl5"; # LIBDIR
```

Comment:
the point of having this on a single line is that the # LIBDIR comment at the end of the line causes the entire line to be replaced with an absolute path upon installation